ores/applications/util.py

```diff
@@ -0,0 +1,27 @@
+import glob
+import os.path
+import yamlconf
+import logging.config
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
+
+
+def build_config(config_dirs=DEFAULT_DIRS,
+                 logging_config="logging_config.yaml"):
+    config_paths = []
+    for directory in config_dirs:
+        config_paths.extend(glob.glob(os.path.join(directory, "*.yaml")))
+    config_paths.sort()
+    config = yamlconf.load(*(open(p) for p in config_paths))
+
+    if logging_config is not None:
+        with open(logging_config) as f:
+            logging_config = yamlconf.load(f)
+            logging.config.dictConfig(logging_config)
+
+    if 'data_paths' in config['ores'] and \
+       'nltk' in config['ores']['data_paths']:
```

Comment:
This would probably be more clear with exception based control flow. Also, it seems fairly common throughout ores to have dynamic imports based on configuration. Is there enough variation in configurations to warrant that?